### DISCUSSION TITLE
Shadcn Discord Server?

### DISCUSSION THREAD

**Discussion by austinm911** [shadcn-ui/ui#1426] [Ideas]
I don't know about you but this lib has been extremely helpful to me. It has helped me go much further than I would have otherwise, alone. 

I have been trying to figure out how to abstract the FormItems of the form component into their own components in a typesafe way and not really finding any help. 

Being in various discord servers has been extremely helpful as I am self-taught so far.

It would be fantastic to see an official Shadcn discord server. It seems like there are a lot of people in the community looking to enhance or add new components (e.g. multiselect combobox). This lib already offers a ton of best practices and I'd love to learn more.

**Comment by austinm911** [comment]
I'll ping the man himself @shadcn

**Comment by jschuur** [comment]
Earlier Twitter discussion on this: https://twitter.com/shadcn/status/1667976991289806849

**Comment by brentarcane** [comment]
It looks like Shadcn never ended up creating one, so I've just created an unofficial Discord channel to dedicated to shadcn/ui - https://discord.gg/RuJwyhaPZ5

Hope to see you on there!

Edit: Updated link.

**Reply by baipai** [reply]
Can you share the link again? thanks!!!

**Reply by ayush0402** [reply]
Hey, is this still active?

**Reply by im-py-dev** [reply]
link plz

**Reply by brentarcane** [reply]
Updated link: https://discord.gg/RuJwyhaPZ5

Accidentally had an expiry set on the original!

**Comment by gislerro** [comment]
@austinm911 have you found a solution to a typesafe abstraction?

**Comment by ghost** [comment]
https://discord.gg/quj7DZFz

hi guys, here is a small unofficial discord channel, let's connect and share awesome things..
webpage is coming soon.. have so many things to share 🚀

**Reply by brentarcane** [reply]
I don't want to steal your thunder, but we do have almost 400 developers on the Discord server mentioned above. You're welcome to join and share your shadcn-ui related work there too! Cheers 😃

**Comment by gzlo** [comment]
this server is still up? can you share de link????

**Reply by AndryOre** [reply]
https://discord.gg/RuJwyhaPZ5

**Reply by brentarcane** [reply]
Thanks @AndryOre! Nearly at 600 users 👍

**Comment by greeneley** [comment]
Hi, Could you share again the invite link ?

**Reply by AndryOre** [reply]
https://discord.gg/RuJwyhaPZ5

**Comment by exodus-tola-mindCoder** [comment]
Introducing: shadcn-remover CLI
A simple, powerful, and user-friendly CLI tool to efficiently remove Shadcn UI components from your project.*


While working on a Next.js project with Shadcn, I added a lot of UI components. Later wanted to remove some of them, I realized there was no easy way to do it.
I checked the official Shadcn docs, browsed through their GitHub repo, and even Googled around — but found no dedicated command or simple solution. Manual deletion was the only option.
That’s when the idea hit me: Why not build a tool for that? — and shadcn-remover CLI was born.

It automates the cleanup process and comes packed with useful features.
By default, it expects your Shadcn UI components to be located at src/components/ui/ (the standard Shadcn structure in  Next.jsV13++).

Check out the full feature list on the GitHub repo and the package README.

Your support through stars and shares is greatly appreciated!

Thanks for using and sharing the shadcn-remover CLI!

NPM: https://www.npmjs.com/package/shadcn-remover
GitHub: https://github.com/exodus-tola-mindCoder/shadcn-remover

#shadcn_remover #ShadcnUI #CLItool #Opensource
